server/authenticate.ts

```diff
@@ -1,5 +1,9 @@
 import config from "./config";
-
+
+/* tslint:disable */
+let bcrypt = require("bcryptjs");

```

Comment:
Yeah it works... I don't understand why at all. When I tried the first time it definitely did not...